README.md

```diff
@@ -176,6 +176,14 @@ Jump to competencies for a particular level:
 
 ## FAQ
 
+### Why these competencies?
+
+We believe a team should prosper and grow _as a team_: that its success is measured by both the quality and throughput of its output _and_ the personal growth and enjoyment of its individual members.
+Strong technical achievements are only half of that.
+Fostering a collaborative community is also necessary for us to be a great place to work.
+
+We've found that these four competency areas capture much of what's worked well for us.
```

Comment:
I would put your last line first, so it reads at a high level as:
- we have data supporting that these are the right choices
- a development team is more than the sum of its technical chops